blocks/core/ff_module/ff_module-tabs-navigation-component/_ff_module-tabs-navigation-component-renderer.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var ReactDOM = require('react-dom');
+var createStore = require('redux').createStore;
+var ReactRedux = require('react-redux');
```

Comment:
These (redux) are only being used in the renderer, right? Just checking in case if we need to update the externals in webpack. In general I think they should only be being used there...